src/js/disability-benefits/526EZ/components/ReviewCollapsiblePage.jsx

```diff
@@ -0,0 +1,158 @@
+    if (path) {
+      newData = _.set([path, index], formData, this.props.form.data);
+    }
+    this.props.setData(newData);
+  }
+
+  focusOnPage = (key) => {
+    const pageDiv = document.querySelector(`#${key}`);
+    focusElement(pageDiv);
+  }
+
+  handleEdit = (key, editing, index = null) => {
+    this.props.onEdit(key, editing, index);
+    this.scrollToPage(`${key}${index === null ? '' : index}`);
+    this.focusOnPage(`${key}${index === null ? '' : index}`);
+  }
+
+  handleSubmit = (formData, key, path = null, index = null) => {
+    // This makes sure defaulted data on a page with no changes is saved
+    // Probably safe to do this for regular pages, too, but it hasn’t been necessary
+    if (path) {
+      const newData = _.set([path, index], formData, this.props.form.data);
+      this.props.setData(newData);
+    }
+
+    this.handleEdit(key, false, index);
+  }
+
+  scrollToTop = () => {
+    scroller.scrollTo(`chapter${this.props.chapterKey}ScrollElement`, window.VetsGov.scroll || {
+      duration: 500,
+      delay: 2,
+      smooth: true,
+    });
+  }
+
+  scrollToPage = (key) => {
+    scroller.scrollTo(`${key}ScrollElement`, window.VetsGov.scroll || {
+      duration: 500,
+      delay: 2,
+      smooth: true,
+    });
+  }
+
+  render() {
+    const { page, chapter, formContext, fullPageKey, form } = this.props;
+    page.pageKey = fullPageKey;
+    const pageState = form.pages[page.pageKey];
+    const editing = pageState.editMode;
+    const ChapterDescription = chapter.reviewDescription;
+    const pageData = form.data;
+
+    const pageContent = (
+      <div className="usa-accordion-content schemaform-chapter-accordion-content" aria-hidden="false">
+        {ChapterDescription &&
+        <ChapterDescription/>}
+        <div key={`${fullPageKey}`} className={'form-review-panel-page'}>
+          <Element name={`${fullPageKey}ScrollElement`}/>
+          {page.schema &&
+          <SchemaForm
+            name={page.pageKey}
+            title={page.reviewTitle || page.title}
+            data={pageData}
+            schema={page.schema}
+            uiSchema={page.uiSchema}
+            hideHeaderRow={page.hideHeaderRow}
+            hideEditButton
+            hideTitle
+            hideBorders
+            pagePerItemIndex={page.index}
+            onBlur={this.props.onBlur}
+            onEdit={() => this.handleEdit(page.pageKey, !editing, page.index)}
+            onSubmit={({ formData }) => this.handleSubmit(formData, page.pageKey, page.arrayPath, page.index)}
+            onChange={(formData) => this.onChange(formData, page.arrayPath, page.index)}
+            uploadFile={this.props.uploadFile}
+            reviewMode={!editing}
+            formContext={formContext}
+            editModeOnReviewPage={page.editModeOnReviewPage}>
+            {!editing ? <div/> : <ProgressButton
+              submitButton
+              buttonText="Update Page"
+              buttonClass="usa-button-primary"/>}
```

Comment:
The prototype depicts a "Cancel" button, but that seems like it would involve disabling autosave, so it was not provided here.